swap/src/bin/swap.rs

```diff
@@ -214,24 +222,7 @@ async fn main() -> Result<()> {
             // Print the table to stdout
             table.printstd();
         }
-        Options::Recover {
-            swap_id,
-            bitcoind_url,
-            monerod_url,
-            bitcoin_wallet_name,
-        } => {
-            let state = db.get_state(swap_id)?;
-            let bitcoin_wallet = bitcoin::Wallet::new(
-                bitcoin_wallet_name.as_ref(),
-                bitcoind_url,
-                config.bitcoin_network,
-            )
-            .await
-            .expect("failed to create bitcoin wallet");
-            let monero_wallet = monero::Wallet::new(monerod_url);
-
-            recover(bitcoin_wallet, monero_wallet, state).await?;
-        }
+        Options::Recover { .. } => todo!("implement this"),
```

Comment:
What is `recover` for? Do you mean, `resume`?

Reply:
This is the terminology that was previously used in the project. I am happy to rename both the CLI option and the function to `resume`.

Reply:
The terminology previously used represented a different feature.

Reply:
my vote goes for resume